docs/2026-06-25_aws_gateway_benchmark/translation/capture.py

```
@@ -0,0 +1,218 @@
+    return obj
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--corpus", default=os.path.join(HERE, "corpus.json"))
+    ap.add_argument("--out", default=os.path.join(HERE, "output", "captures.json"))
+    ap.add_argument("--gateways", default=",".join(ORDER))
+    args = ap.parse_args()
+
+    gateways = [g for g in args.gateways.split(",") if g in GATEWAYS]
+    corpus = json.load(open(args.corpus, encoding="utf-8"))
```

Ruff (0.15.18)

[warning] Use a context manager for opening files

(SIM115)